### ISSUE TITLE
Gather and Sort Resources for Curriculum Production

### ISSUE THREAD

**Issue by LauraHilliger** [mozilla/learning-networks#162]
This work started in issues #17 and #105. I've created a new issue to streamline conversation around the new sorting mechanism.

I've been collecting Curricular Resources and sorting against the [Web Literacy Map](https://teach.mozilla.org/teach-like-mozilla/web-literacy/) and Open Web Leadership map ([in parallel production](http://www.zythepsary.com/techie/open-fluency/)). 

**[This is the Master Curriculum Sorting Sheet](https://docs.google.com/a/zythepsary.com/spreadsheets/d/1VHj-89BhI4HtUQfXOW9jKlSGLx-FXV9Er1G86HKvtZU/edit#gid=0)**

These two sorting methods represent the **_2 Streams of Curriculum we produce - the Web Literacy Stream and the Teach Like Mozilla Stream.**_

Collected Resources should serve as a baseline for new curriculum production. For Production Queue, please see [the Production Queue](http://mozilla.github.io/webmaker-curriculum/) :+1:

**Comment by thornet** [comment]
cc @omnignorant

**Comment by LauraHilliger** [comment]
@omnignorant we can use this issue and spreadsheet to crosscheck the Club activities with the web literacy skills as well. Maybe just make a new sheet / or a new column to list which skill is covered in which activity? Let's discuss tomorrow :) In the meantime, feel free to jump into this one.

**Comment by ldecoursy** [comment]
@chadsansing Do you have relevant info from this ticket for plans moving forward? If so we can close.

**Comment by chadsansing** [comment]
I have the sorting doc bookmarked.

**Comment by ldecoursy** [comment]
OK thanks - closing!
